spqr/profile_spqr.py

```diff
@@ -0,0 +1,238 @@
+import numpy as np
+import torch
+import spqr_cuda
+import time
+import os
+
```

Comment:
What is this profiling and how it is dffer from bench_spqr.py?

Reply:
Removed.